hack/build.sh

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env bash
+#
+# To push to your own registry, override the REGISTRY and NAMESPACE env vars,
+# i.e:
+#   $ REGISTRY=quay.io NAMESPACE=yourusername ./hack/build.sh
+#
+# REQUIREMENTS:
+#  * a valid login session to a container registry.
+#  * `docker`
+#  * `jq`
+#  * `yq`
+#  * `opm`
+#  * `skopeo`
+
+set -eu -o pipefail
+
+declare -r OPERATOR_NAME='observability-operator'
+declare -r REGISTRY=${REGISTRY:-'quay.io'}
+declare -r NAMESPACE=${NAMESPACE:-'rhobs'}
+declare -r TAG=${TAG=$1}
+declare -r CSV_PATH=${CSV_PATH:-'bundle/manifests/observability-operator.clusterserviceversion.yaml'}
+declare -r ANNOTATIONS_PATH=${ANNOTATIONS_PATH:-'bundle/metadata/annotations.yaml'}
+
+cleanup() {
+	# shellcheck disable=SC2046
+	if [ -x $(command -v git >/dev/null 2>&1) ]; then
+		git checkout "${CSV_PATH}" >/dev/null 2>&1
+		git checkout "${ANNOTATIONS_PATH}" >/dev/null 2>&1
+	fi
+}
+
+trap cleanup EXIT
+
+# prints pre-formatted info output.
+info() {
+	echo "INFO $(date '+%Y-%m-%dT%H:%M:%S') $*"
+}
+
+# prints pre-formatted error output.
+error() {
+	>&2 echo "ERROR $(date '+%Y-%m-%dT%H:%M:%S') $*"
+}
+
+digest() {
+	local -n ret=$2
+	IMAGE=$1
+	podman pull "${IMAGE}"
```

Comment:
is this specific to podman? could we use just the same CONTAINER_RUNTIME env var as above for this?